packages/deploy-config-sub-generator/src/types/index.ts

```diff
@@ -0,0 +1,103 @@
+import type Generator from 'yeoman-generator';
+import type { AppWizard } from '@sap-devx/yeoman-ui-types';
+import type { ApiHubConfig } from '@sap-ux/cf-deploy-config-writer';
+import type { ServiceProvider } from '@sap-ux/axios-extension';
+import type { BackendSystem } from '@sap-ux/store';
+import type { Destination } from '@sap-ux/btp-utils';
+import type { LogWrapper } from '@sap-ux/fiori-generator-shared';
+
+export interface DeployConfigOptions extends Generator.GeneratorOptions {
+    /**
+     * The target deployment
+     */
+    target?: string;
+
+    /**
+     * Instance of the AppWizard
+     */
+    appWizard: AppWizard;
+    /**
+     * Whether the deploy config should be launched as a sub generator
+     * All prompts (CF & ABAP) wil be combined into one step
+     */
+    launchDeployConfigAsSubGenerator?: boolean;
+    /**
+     * Host URL passed from the generator
+     */
+    appGenServiceHost: string;
+    /**
+     * Service path passed from the generator
+     */
+    appGenServicePath: string;
+    /**
+     * Client passed from the generator
+     */
+    appGenClient: string;
+    /**
+     * The connected system from the generator
+     */
+    connectedSystem?: {
+        serviceProvider: ServiceProvider;
+        backendSystem?: BackendSystem;
+        destination?: Destination;
+    };
+    /**
+     * Whether the system is a cloud system
+     */
+    scp?: boolean;
```

Comment:
This property is no longer being passed and should be considered deprecated. It can be calculated internally if required.

Reply:
Done